core/icons/comment_icon.ts

```diff
@@ -110,6 +110,7 @@ export class CommentIcon extends Icon implements IHasBubble, ISerializable {
       },
       this.svgRoot,
     );
+    dom.addClass(this.svgRoot, 'blockly-icon-comment');
```

Comment:
```suggestion
    dom.addClass(this.svgRoot!, 'blockly-icon-comment');
```

You can add an `!` here (and below) to deal with the nullability. There are more type safe ways we could deal with that (e.g. using a different var to track whether the icon has been initialized or not). But this is fine because `initView` guarantees that `svgRoot` will be constructed.